doc/source/performance/threadpool.rst

```diff
@@ -4,41 +4,44 @@
  Thread Pool
 =============
 
-|MySQL| executes statements using one thread per client connection. Once the number of connections increases past a certain point performance will degrade.
+|MySQL| assigns one thread for each client connection which executes every statement for that connection. As the number of connections increases and each connection executes a series of statements, the performance deteriorates. Threads cost RAM, therefore increasing the thread number can cause other issues, such as resource contention, or excessive context switching.
 
-This feature enables the server to keep the top performance even with a large number of client connections by introducing a dynamic thread pool. By using the thread pool server would decrease the number of threads, which will then reduce the context switching and hot locks contentions. Using the thread pool will have the most effect with ``OLTP`` workloads (relatively short CPU-bound queries).
+A dynamic thread pool enables the server to maintain top performance with a large number of clients. A thread pool decreases the number of threads on the server, which also reduces the context switching and hot locks contentions. A thread pool has the most effect with ``OLTP`` workloads (relatively short CPU-bound queries).
 
-In order to enable the thread pool variable :variable:`thread_handling` should be set up to ``pool-of-threads`` value. This can be done by adding: ::
+In order to enable the thread pool, the variable :variable:`thread_handling` should be set up to ``pool-of-threads`` value. This can be done by adding this setting to the |MySQL| configuration file :file:`my.cnf`: ::
 
  thread_handling=pool-of-threads
 
-to the |MySQL| configuration file :file:`my.cnf`.
+.. seealso::
+
+   |MySQL| Documentation
+   `thread-handling <https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_thread_handling>`_
 
-Although the default values for the thread pool should provide good performance, additional `tuning <https://kb.askmonty.org/en/threadpool-in-55/#optimizing-server-variables-on-unix>`_ can be performed with the dynamic system variables described below.
+Although the default values for the thread pool should provide good performance, additional tuning can be performed with the dynamic system variables described in system-variables_.
```

Comment:
I'd rewrite the second part of the sentence in active voice : "..., you can additionally tune thread pool with the dynamic system variables described in  system-variables."

Reply:
Rewritten.